android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1365,6 +1366,9 @@ public static boolean setKeyboard(Context context, int position) {
   public static void switchToNextKeyboard(Context context) {
     int index = KeyboardController.getInstance().getKeyboardIndex(KMKeyboard.currentKeyboard());
     index++;
+    if (index >= KeyboardController.getInstance().get().size()) {
+      index = 0;
+    }
```

Comment:
I can split your other suggestions into separate PRs. This one is just adding robustness, since KeyboardController will automatically revert to index 0.

I'd like to keep this one on this PR though, since we don't assign `switchToNextKeyboard`

Reply:
Though maybe I can bundle this with the registerAssociatedLexicalModel call onto the separate PR and temporarily assign the globe key to switchToNextKeyboard